monorepo-migration/sync-focus.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
```

Comment:
Nit: let's rename the file in a separate commit (which could be part of this PR still). It'll make the diff easier to review.

Reply:
Just realized and did that :) I will remove this sync-focus.sh and rename it later on